ci-operator/config/openshift-assisted/cluster-api-provider-openshift-assisted/openshift-assisted-cluster-api-provider-openshift-assisted-master.yaml

```diff
@@ -67,6 +73,29 @@ tests:
   steps:
     cluster_profile: packet-assisted
     workflow: assisted-ofcir-baremetal-capi
+- as: periodic-version-discovery
+  cron: 0 */12 * * *
+  steps:
+    env:
+      DRY_RUN: "true"
+    test:
+    - ref: assisted-capi-version-discovery
+- as: periodic-test-runner
+  cron: 0 */6 * * *
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      DRY_RUN: "true"
+      TEST_TARGET: snapshots-test
+    workflow: assisted-ofcir-baremetal-capi
+- as: tags-reconciliation
```

Comment:
This should be triggered only when versions.yaml changed, right?
No need to trigger it on each PR